research/classification_layer.py

```diff
@@ -0,0 +1,67 @@
+#%%
+
+# Have to create this file in order to call the unaccepted model from research
+from pento.data_processing import process_data
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+from matplotlib import image as mpimg
+import research.start
+import torch
+import torch.nn as nn
+from torch.autograd import Variable
+from sklearn.model_selection import train_test_split
+from torch.utils.data import TensorDataset, DataLoader
+import torch.nn.functional as F
+from sklearn.metrics import accuracy_score, f1_score
+import torchvision.transforms as T
+from PIL import Image
+import os
+from numpy import expand_dims
+import pretrainedmodels as pm
+
+#%%
+# In this file I'll define the classification model use for this project
+# For multiclassification I'll use a fully-connected nerual network with a
+# configurable number of layers, and size of layers.
+
+
+class Net(nn.Module):
+    def __init__(self, in_features, list_of_weights, num_classes=12):
+        super(Net, self).__init__()
+        self.sig = nn.Sigmoid()
+        self.input_size = (
+            in_features  # Change this to the dimension of the output from the resnet.
+        )
+        counter = 0
+        layer = (
+            in_features  # Cahnge this to the dimension of the output from the resnet.
+        )
+
+        layers_array = []
+
+        # layers_array.append(nn.Flatten())
+
+        for i in list_of_weights:
+            batchnomr_layer = nn.BatchNorm1d(layer)
+            layers_array.append(batchnomr_layer)
+            linear_layer = nn.Linear(layer, list_of_weights[counter])
+            layers_array.append(linear_layer)
+            activation = nn.ReLU6()
+            layers_array.append(activation)
+
+            layer = list_of_weights[counter]
+            counter += 1
+
+        self.layers = nn.Sequential(*layers_array)
+        self.output = nn.Linear(layer, num_classes)
+
+        # x represents our data
+
+    def forward(self, x):
+        y = x
+        for layer in self.layers:
+            y = layer(y)
```

Comment:
Since you packed the layers in a Sequential, you can simply call
```suggestion
        y = self.layers(y)
```

Reply:
Cool I didn´t know this!